src/risedevtool/src/bin/risedev-dev.rs

```diff
@@ -336,6 +337,13 @@ fn task_main(
     Ok((stat, log_buffer))
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+struct Input {
+    config_path: Option<String>,
+    steps: Vec<ServiceConfig>,
+}
```

Comment:
The generated JSON will be directly deserialized into this struct, then executed by `risedev-dev`.